lib/ddtrace/contrib/httparty/configuration/settings.rb

```diff
@@ -0,0 +1,24 @@
+require 'ddtrace/contrib/configuration/settings'
+require 'ddtrace/contrib/httparty/ext'
+
+module Datadog
+  module Contrib
+    module HTTParty
+      module Configuration
+        # Custom settings for the HTTParty integration
+        class Settings < Contrib::Configuration::Settings
+          option  :analytics_enabled,
+                  default: -> { env_to_bool(Ext::ENV_ANALYTICS_ENABLED, false) },
+                  lazy: true
+
+          option  :analytics_sample_rate,
+                  default: -> { env_to_float(Ext::ENV_ANALYTICS_SAMPLE_RATE, 1.0) },
+                  lazy: true
```

Comment:
We've just introduced a new configuration DSL syntax for `option`s, would you mind refactoring these to look similar to this example? https://github.com/DataDog/dd-trace-rb/blob/18300bd2c645c213c22ade2fd66b5bc6151be450/lib/ddtrace/contrib/racecar/configuration/settings.rb#L10-L20

Reply:
I'm running into some trouble with this change. When running my tests, `o` is always `nil`.

Reply:
It seems like you need to rebase your branch with the latest master :)